shiva/src/main/scala/nl/lumc/sasc/biopet/pipelines/shiva/Shiva.scala

```diff
@@ -287,7 +302,12 @@ class Shiva(val parent: Configurable)
 
       annotation.foreach { toucan =>
         toucan.outputDir = new File(outputDir, "annotation")
-        toucan.inputVcf = vc.finalFile
+        toucan.inputVcf = vc.finalFile.get
+        add(toucan)
+      }
+      annotationForSomaticVariants.foreach { toucan =>
+        toucan.outputDir = new File(outputDir, "annotation")
```

Comment:
Not sure but also not sure that those files names are always correct. Like the summary .out and .done files are the same name, this causing issues in the graph